src/pages/learn/index.mdx

```diff
@@ -1,42 +1,46 @@
+import { Callout } from "nextra/components"
+
 # Introduction to GraphQL
 
-> Learn about GraphQL, how it works, and how to use it. Looking for documentation on how to build a GraphQL service?
-> There are libraries to help you implement GraphQL in [many different languages](/code). For an in-depth learning experience
-> with practical tutorials, see [the available training courses](/community/resources/training-courses).
+<p className="learn-subtitle">Learn about GraphQL, how it works, and how to use it</p>
+
+GraphQL is a query language for your API, and a server-side runtime for executing queries using a type system you define for your data. The [GraphQL specification](https://spec.graphql.org/) was open-sourced in 2015 and has since been implemented in a variety of programming languages. GraphQL isn't tied to any specific database or storage engine—it is backed by your existing code and data.
 
-GraphQL is a query language for your API, and a server-side runtime for executing queries using a type system you define for your
-data. GraphQL isn't tied to any specific database or storage engine and is instead backed by your existing code and data.
+<Callout type="info">If you're already familiar with GraphQL and would like to read documentation on how to build a GraphQL service, then there are libraries to help you implement GraphQL in [many different languages](/community/tools-and-libraries/?tags=server). There are also many libraries available that allow [client applications to query existing GraphQL APIs](/community/tools-and-libraries/?tags=client).</Callout>
```

Comment:
Should we mention about GraphQL.js being the official reference implementation maintained by the GraphQL foundation?

Reply:
@benjie Do you want to call out graphql.js specifically here?

Reply:
I don't think we need to do that here, but our first mention of a "reference implementation" should hyperlink to GraphQL.js